testsuite/features/step_definitions/xmlrpc_common.rb

```diff
@@ -479,7 +480,14 @@
   end
 
   actions.each do |action|
-    scdrpc.cancel_actions([action['id']])
+    puts "\t- Try to cancle \"" + action['name'] + '" action'
```

Comment:
puts "\t- Try to cancel \"#{action['name']}\" action"

Reply:
done